frontend/src/Commons/Forms/InputModules/Calculate/Calculate.tsx

```diff
@@ -0,0 +1,45 @@
+/**
+ * @summary Farm Information Input Module
+ * @description A module with input fields to be used in a Form Module
+ * @author @GDamaso
+ */
+import Button from '@Commons/Button/Button.tsx';
+import InputModuleInterface from 'src/Interface/InputModuleinterface';
+import { faCalculator } from '@fortawesome/free-solid-svg-icons';
+import { StyledFarmInfo, StyledButtonContainer } from './Calculate.style';
+
+const FarmInfoComponent = () => (
+  <StyledFarmInfo>
+    <label htmlFor="farmName">
+      Name
+      <br />
+      <input type="text" name="farmName" />
+    </label>
+    <label htmlFor="farmYear">
+      Year
+      <br />
```

Comment:
please write better CSS than relying on `br` tags to break lines for you 
https://blog.hubspot.com/website/html-line-break